pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
+
+`pngquant {{path/to/file.png}}`
+
+- Compress a PNG and override original:
+
+`pngquant --ext .png -f {{path/to/file.png}}`
+
+- Try to compress a PNG with custom quality (skip if below the min value):
+
+`pngquant -Q {{0-100}} {{path/to/file.png}}`
+
+- Compress a PNG with the number of colors reduced to 64:
+
+`pngquant {{64}} {{path/to/file.png}}`
+
+- Compress a PNG and skip if the file is larger than the original:
+
+`pngquant --skip-if-larger {{path/to/file.png}}`
+
+- Compress a PNG and remove metadata:
+
+`pngquant --strip {{path/to/file.png}}`
+
+- Compress a PNG and save to the given path:
+
+`pngquant {{path/to/file.png}} -o {{path/to/file.png}}`
+
+- Compress a PNG and show progress:
```

Comment:
There are multiple fixes available: please chose the most appropriate one.

```suggestion
- Compress a specific PNG and show progress:
```

Optional change: `a specific` may sound better.

```suggestion
- Compress a specific .png and show progress:
```

Optional change: you can use `.ext` extension format instead of `EXT` one.